include/aspect/material_model/visco_plastic.h

```diff
@@ -276,11 +235,10 @@ namespace aspect
 
       private:
 
-        double min_strain_rate;
-        double ref_strain_rate;
-        double min_visc;
-        double max_visc;
-        double ref_visc;
+        /**
+         * Object for computing viscosities.
```

Comment:
I think the documentation for this object should be expanded quite a bit. While one the functions does compute the viscosity, this object also points to functions for additional material model outputs, the Newton derivatives, etc. Perhaps a more accurate single sentence description would be 'Object for rheology operations'. However, my preference would be to have an expanded description.